### DISCUSSION TITLE
Download tracking for assets like PDFs

### DISCUSSION THREAD

**Discussion by rufuspollock** [flowershow/flowershow#672] [General and Help]
How would this work? How would it interact with remark-wiki-link-plus?

**Comment by joesmusings** [comment]
Users could do this given ability to install Google Tag Manager via the config file, and set up custom tags & goals via Google Analytics!

**Comment by rufuspollock** [comment]
Is there a way for folks to do this already just by using google analytics? I think so ... so closing as fixed. Maybe more of a howto for the discussion forum.
